.github/workflows/continuous-integration.yml

```diff
@@ -348,7 +412,7 @@ jobs:
       fail-fast: false
       max-parallel: 8
       matrix:
-        ci_node_index: [0, 1, 2, 3, 4, 5, 6, 7]
+        ci_node_index: ${{ fromJson(needs.cypress-tests-prep.outputs.ci_node_index) }}
```

Comment:
`cypress-tests-prep` is not running on master in the latest commit. What will `ci_node_index` be for master build?

Reply:
`ci_node_index` is set manually in this step (the step after that one) on merge to master:

```
      - name: Run Cypress tests on merge to master
        if: ${{ github.ref == 'refs/heads/master' }}
        run: node script/github-actions/run-cypress-tests.js
        timeout-minutes: 30
        env:
          # CYPRESS_BASE_URL: http://localhost:3001
          CYPRESS_CI: true
          STEP: ${{ matrix.ci_node_index }}
```